doc/sphinx/source/recipes/recipe_extreme_events.rst

```diff
@@ -0,0 +1,134 @@
+Extreme Events Indices - Computation of ETCCDI extreme indices and plotting
```

Comment:
Add an entry in to doc/sphinx/source/recipes/index.rst, now this does not get included.

Reply:
Ok, done.